### DISCUSSION TITLE
Bug in Dockge Agent

### DISCUSSION THREAD

**Discussion by EdgarM73** [louislam/dockge#432] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

I have set up a second instance but couldnt use port 5001 for it.

When trying to connect the agent to this new instance with 5101 i received an error message "could not connect " or so.

Than i change to port 5001 and it worked from scratch.

### 👟 Reproduction steps

using port different to 5001

### 👀 Expected behavior

to be able to connect to agent

### 😓 Actual Behavior

not able to connect 

### Dockge Version

1.4.2

### 💻 Operating System and Arch

debian docker

### 🌐 Browser

chrome

**Comment by huzzyz** [comment]
How are you specifying the ports? Working fine here. Share your docker-compose.yml
